interface/patient_file/summary/rx_left.php

```diff
@@ -41,7 +41,7 @@
 </tr>
 <tr>
 <td>
-<?php if ($GLOBALS['rx_show_drug-drug']){ ?>
+<?php if ($GLOBALS['weno_rx_enable']){ ?>
```

Comment:
No, It was not in the globals and because it was missing I did not put it back. I just changed it. I can put it back if you that is what you want.